loops_lab.js

```diff
@@ -0,0 +1,313 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+let num = 20
+while(num > 0) {
+  console.log(num);
+  num--
+}
+console.log('-----------')
+for(i = 20; i > 0; i--){
+  console.log(i)
+}
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+console.log('-----------')
+
+let max = 0
+while(max >= 0 && max <= 10 ){
+  if(max % 2 === 0){
+    console.log(max + ' is even.')
+  } else {
+    console.log(max + ' is odd.')
+  }
+  max++
+}
+
+console.log('-----------')
+
+for (maxx = 0; maxx >= 0 && maxx <= 10; maxx++ ){
```

Comment:
Same thing here as well. No need for two conditions inside for loops if once condition would suffice the code.